regression/ansi-c/float_constant1/main.c

```diff
@@ -21,9 +21,21 @@ STATIC_ASSERT(__builtin_types_compatible_p(_Float64, __typeof(1.0f64)));
 STATIC_ASSERT(__builtin_types_compatible_p(_Float128, __typeof(1.0f128)));
 STATIC_ASSERT(__builtin_types_compatible_p(_Float32x, __typeof(1.0f32x)));
 STATIC_ASSERT(__builtin_types_compatible_p(_Float64x, __typeof(1.0f64x)));
+// f128x should be supported by GCC >= 7 (and for C++ in GCC >=13), but there
+// are not current GCC ports that actually support such types
```

Comment:
not -> no
What is meant by a "gcc port" here?

Reply:
Fixed.